backend/src/main/java/saci/domain/model/Knowledge.java

```diff
@@ -0,0 +1,31 @@
+package saci.domain.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Data;
+
+@Data
+@Entity
+@Table(name = "knowledge")
+public class Knowledge {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name")
+    private String name;
+
+    @Column(name = "role_id")
+    private long roleId;
```

Comment:
Long and long are different number implementations, for example the Long can be null and long can't, so as we still don't have the "levels" table we should keep the Long type in all to keep consistency.